inst/rmarkdown/templates/trb/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,97 @@
+  
+keywords: ["Transportation", "Travel Behavior"]
+bibliography: mybibfile.bib
+biblio-style: unsrtnat
+linenumbers: true
+#numbersections: true
+output: rticles::trb_article
+wordcount: 1176
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+Transportation Research Board Annual Meetings
+==========================
+
+TRB has greatly simplified the submissions to its system, meaning that this
+template is no longer strictly required. However, there is a typical format
+that most people still use by default, and this template implements that change.
+
+The automatic word count is not yet engaged in the rticles version of the
+template. Set the word count using the yaml header. You can reference figures
+with \@ref(fig:figure-example).
+
+
+
+\subsection{History}
+David Pritchard posted the original versions of this template in 2009 and updated it in 2011, soon after TRB began allowing PDF submissions. Gregory Macfarlane made significant adaptations to it in March 2012, allowing for Sweave integration and automatic word and table counts. Ross Wang automated the total word count and made some formatting modifications in July 2015. Version 2.1.1 has been made available on GitHub in January, 2016.  Version 3.1 has been made available on Github (\url{https://github.com/chiehrosswang/TRB_LaTeX_rnw}) in June, 2017.  Versions 2.1.1 Lite and 3.1 Lite were made available on GitHub (\url{https://github.com/chiehrosswang/TRB_LaTeX_tex}) in June, 2017 for users who do not need R and Sweave functions provided in the original verions.  
+Gregory Macfarlane created the `rticles` template in 2021.
+
+# Features
+The template has a number of features that enable quick and painless manuscript authoring.
+
+## Title Page
+The standard \LaTeX `\maketitle` command is not very versatile, so we have replaced it with a `verb1titlepage` environment. This means that the writers will be required to manually enter spacings based on the number of contributors, but the current settings (12pt between authors, 36pt before, and 60pt after them) seems to work well. 
+
+Near the bottom of the title page, TRB requires a count of the manuscript's words, figures, and tables. This template generates these counts automatically. The figure and table counts are simply pulled from the \LaTeX\ counters using the \verb1totcount1 package. The word count feature is not as straight-forward, as it utilizes a call to the system command \verb1texcount1. Thus to compile the document writers must enable \verb#\write18# in their \verb1pdflatex1 call.
```

Comment:
Does this mean `wordcount` variable do not need to be provided ? 

Is a package required for `\verb1totcount1` to work ?